ChopChop/Views/OnlineRecipes/OnlineRecipeView.swift

```diff
@@ -4,86 +4,142 @@ struct OnlineRecipeView: View {
     @ObservedObject var viewModel: OnlineRecipeViewModel
 
     var body: some View {
-        VStack {
-            Image(uiImage: viewModel.image)
+        VStack(spacing: 0) {
+            userBar
+            recipeImage
+            averageRating
+            if viewModel.isShowingDetail {
+                recipeDetails
+            }
+            Divider()
+            showDetailBar
+        }
+    }
+
+    var userBar: some View {
+        HStack {
+            Image("default-user")
                 .resizable()
                 .scaledToFill()
-                .frame(height: 200)
-                .clipped()
+                .frame(width: 50, height: 50)
+                .clipShape(Circle())
+            Text(viewModel.creatorName)
```

Comment:
If its the user's own recipe, would it be strange if we display his own name?

Reply:
I think this is fine (Instagram does this too)